### PR TITLE
Internal testing with multiple endpoints

### PR DESCRIPTION
**Pull request by bryanmikaelian**
This is an small internal test to see if we can support measurements to multiple API endpoints based on config settings. 

There isn't any plans to merge this to master

### CODE REVIEW

**Comment by bryanmikaelian** [comment]
https://my-stg.appoptics.com/s/metrics/api.auth.time?q=api

File: integ/test_librato.py [hunk 1]
```diff
@@ -499,3 +504,20 @@ def test_measurements_with_host(self):
...
+        })
+
+        expected_ao_output = [{
+            "name":         "active_sessions",
+            "time":         1401577507,
+            "value":        1.0,
+            "tags":         {}
+        }]
+
+        k = "active_sessions"
```
Comment (jderrett) [inline integ/test_librato.py:521]: I think this doesn't get used?

Comment (bryanmikaelian) [reply integ/test_librato.py]: Huh weird.

**Comment by bryanmikaelian** [comment]
Closing this out due to a different approach

**Pull request closed without merging**